### PR TITLE
Azure-IoT-Sdk for C release 2020-12-09

### PR DESCRIPTION
**Pull request by ewertons**
**Describe the pull request**

- What does your PR fix? Fixes #
It updates references of our ports to reflect our new release done on 12/09/2020.

- Which triplets are supported/not supported? Have you updated the CI baseline?

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

### CODE REVIEW

File: ports/azure-iot-sdk-c/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: azure-iot-sdk-c
-Version: 2020-07-19
+Version: 2020-12-09
 Port-Version: 1
```
Comment (NancyLi1013) [inline ports/azure-iot-sdk-c/CONTROL:3]:
```suggestion
```
For a new version update, this field is not needed.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @ewertons 

Thanks for your PR.

Have you tested these features in this PR?

**Comment by ewertons** [comment]
Thanks @NancyLi1013 ! Fixed. Yes, we've tested these changes.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
LGTM now, thanks for your PR @ewertons.

**Comment by BillyONeal** [comment]
Looks great, thanks!

**Pull request merged**